src/api/init.ts

```diff
@@ -0,0 +1,63 @@
+import Fastify, { FastifyPluginAsync } from 'fastify';
+import { TypeBoxTypeProvider } from '@fastify/type-provider-typebox';
+import { FtRoutes } from './routes/ft';
+import { NftRoutes } from './routes/nft';
+import { PgStore } from '../pg/pg-store';
+import FastifyCors from '@fastify/cors';
+import FastifySwagger, { SwaggerOptions } from '@fastify/swagger';
+import { StatusRoutes } from './routes/status';
+import FastifyMetrics from 'fastify-metrics';
+import { Server } from 'http';
+
+export const ApiSwaggerOptions: SwaggerOptions = {
+  openapi: {
+    info: {
+      title: 'Stacks Token Metadata Service',
+      description:
+        'A microservice that indexes metadata for every SIP-009, SIP-010, and SIP-013 Token in the Stacks blockchain and exposes it via REST API endpoints.',
+      version: '0.0.1',
```

Comment:
Is this version exposed anywhere? Could possibly grab it from the package.json (e.g. `require('./package.json').version`) or env var / git tag.